app.js

```diff
@@ -0,0 +1,39 @@
+const os = require('os');
+const fs = require('fs');
+const childProcess = require('child_process');
+
+const commands = {
+  unix: 'ps -A -o %cpu,%mem,comm | sort -nr | head -n 1',
+  windows: 'powershell "Get-Process | Sort-Object CPU -Descending | Select-Object -Property Name, CPU, WorkingSet -First 1 | ForEach-Object { $_.Name + \' \' + $_.CPU + \' \' + $_.WorkingSet }"'
+};
+
+const command = os.platform() === 'win32' ? commands.windows : commands.unix;
+
+const saveData = (data) => {
+  setInterval(() => {
+    const time = new Date().getTime();
+    const newLine = `${time}: ${data}`;
+    fs.appendFile('activityMonitor.log', newLine, (err) => {
+      if (err) throw err;
+      console.warn('Updated!');
+    });
+  }, 60 * 1000);
+};
+
+const execProcess = (cmd) => {
+  childProcess.exec(cmd, (error, stdout) => {
+    console.clear();
+    console.log(stdout);
+    saveData(stdout);
+  });
+};
+
+setInterval(() => {
+  execProcess(command);
+}, 100);
+
```

Comment:
add another setInterval here only for `saveData` function call:

```suggestion
setInterval(() => {
  execProcess(command);
}, 100);

setInterval(() => {
  saveData(command);
}, 60 * 1000)

```

didn't test it, but it should work